sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/ConnectionStringParser.cs

```diff
@@ -51,7 +51,7 @@ internal static string ParseTokenAndReturnValue(string connectionString, string
         ///
```

Comment:
To disambiguate this type and the (now shared)  Event Hubs parser.... maybe we should consider tweaking the name.   Something like `ConnectionStringTokenParser` or similar, since it's job is to hunt for a particular token rather than generally extract a bunch of well-known information?